src/modules/createNewGame.js

```diff
@@ -0,0 +1,34 @@
+import saveGameID from './saveGameID.js';
+
+const postData = { name: 'Need for Speed' };
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+const createNewGame = async () => {
+  const requestOptions = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(postData),
+  };
+
+  try {
+    const response = await fetch(url, requestOptions);
+
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+
+    const data = await response.json();
+    const { result } = data;
+    const idMatch = result.match(/ID:\s*(\w+)/);
+    if (idMatch && idMatch[1]) {
+      const extractedId = idMatch[1];
+      saveGameID(extractedId);
+    }
+  } catch (error) {
+    // return error;
+  }
+};
```

Comment:
- [ ] Please consider removing the commented code from our codebase to maintain a clean and professional code repository.

- [OPTIONAL] I wanted to suggest a slight improvement in our code. Instead of leaving the `catch` block empty, consider throwing an error within it. This way, you can proactively handle the error at a later point in the code.